src/components/tabs/__internal__/tab-title/tab-title.style.js

```diff
@@ -14,9 +14,7 @@ const StyledTitleContent = styled.span`
     position,
     noLeftBorder,
     noRightBorder,
-    isTabSelected,
     hasHref,
-    error,
```

Comment:
I think you should also remove these props from <StyledTitleContent ...> in src/components/tabs/__internal__/tab-title/tab-title.component.js

Reply:
They are actually used still but destructured later in the file, I'll refactor it so it's done at the top level